File: app/java/com/longtoast/bilbil_api/dto/NaverTokenRequest.java
package com.longtoast.bilbil_api.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class NaverTokenRequest {
    // 안드로이드에서 보낸 필드명 (accessToken)과 일치해야 합니다.
    private String accessToken;
}

File: app/src/main/java/com/longtoast/bilbil/dto/NaverTokenRequest.kt
// java/com/longtoast/bilbil/dto/NaverTokenRequest.kt
package com.longtoast.bilbil.dto

// Naver Access Token을 서버로 보냅니다.
data class NaverTokenRequest(
    val accessToken: String // 서버 DTO의 필드명과 일치해야 함
)